projects/ion/src/lib/utils/datePicker.spec.ts

```diff
@@ -0,0 +1,143 @@
+import { Day } from '../picker/core/day';
+import {
+  getFormattedDate,
+  getInitialDate,
+  isBetweenRange,
+  isSameDate,
+  isSameDay,
+  isToday,
+} from './datePicker';
+
+const today = new Date('2024-01-10T03:00:00.000Z');
+jest.useFakeTimers('modern').setSystemTime(today);
+
+describe('DatePicker', () => {
+  let result: boolean;
+  describe('isToday', () => {
+    let currentDate: Day;
+    let lang: string;
+
+    it('should return true if day is today', () => {
+      currentDate = new Day(today);
+      lang = 'pt-BR';
+
+      result = isToday(currentDate, lang);
+
+      expect(result).toBe(true);
+    });
+    it('should return false if day is not today', () => {
+      currentDate = new Day(new Date('2024-01-03'));
+      lang = 'pt-BR';
+
+      result = isToday(currentDate, lang);
+
+      expect(result).toBe(false);
+    });
+  });
```

Comment:
```suggestion
  describe('isToday', () => {
    let currentDate: Day;
    const lang = 'pt-BR';

    it('should return true if day is today', () => {
      currentDate = new Day(today);
      result = isToday(currentDate, lang);
      expect(result).toBe(true);
    });
    
    it('should return false if day is not today', () => {
      currentDate = new Day(new Date('2024-01-03'));
      result = isToday(currentDate, lang);
      expect(result).toBe(false);
    });
  });
```

lang not is modified, so we can use const